GSoC-Participants.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: GSoC participants
+---
+
+This document collects the list of contributors who've contributed
+to Git via GSoC.
+
+<!-- [ [project]() ] [ [final report]() ] [ [blog]() ] -->
+
+### 2023
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2023/organizations/git)
```

Comment:
From _posts/2023-06-30-edition-100.markdown:

    - Kousik Sanagavarapu will work on the "[Unify ref-filter formats with other pretty formats](https://summerofcode.withgoogle.com/programs/2023/projects/rck3kmq2)"
      project. Kousik will be co-mentored by Hariom Verma and Christian Couder.
      Some details about his work on the project could be found in [his blog](https://five-sh.github.io/blog).

    - Shuqi Liang will work on the "[More Sparse Index Integrations](https://summerofcode.withgoogle.com/programs/2023/projects/Rkbc1Abe)"
      project. Shuqi will be mentored by Victoria Dye. Further details about
      the project and updates on progress can be found on [Shuqi's blog](https://cheskaqiqi.github.io/tags/GSoC/).